README.md

```diff
@@ -25,6 +25,8 @@ S3 Cache for GitHub Actions supports builds on Linux, Windows and MacOS.
     aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     aws-region: us-east-1 # Or whatever region your bucket was created
     bucket: your-bucket
+    s3class: STANDARD # It's STANDARD by default. It can be either STANDARD, 
```

Comment:
Can you use `kebab-case`, please? 
About default value, you can use [default field](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs) for that.

Reply:
I believe `camelCase` is a better approach since it's a matter of syntax standards.